### PR TITLE
ras.py: changes made to eliminate ofpathname error

### PR DESCRIPTION
**Pull request by debade33**
changes made to eliminate ofpathname error

Signed-off-by: Debaleena De <debade33@in.ibm.com>

### CODE REVIEW

**Comment by debade33** [comment]
before patch
JOB ID     : 2453f79d6aff8f39a46c2a262799f9f0e14572cc
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-12-31T14.12-2453f79/job.log
 (01/12) ras.py:RASTools.test1_set_poweron_time: PASS (2.61 s)
 (02/12) ras.py:RASTools.test2_sys_ident_tool: PASS (2.52 s)
 (03/12) ras.py:RASTools.test3_lsmcode: PASS (2.83 s)
 (04/12) ras.py:RASTools.test4_drmgr: PASS (16.03 s)
 (05/12) ras.py:RASTools.test5_lsprop: PASS (2.24 s)
 (06/12) ras.py:RASTools.test6_lsslot: PASS (2.51 s)
 (07/12) ras.py:RASTools.test7_lsvio: PASS (2.07 s)
 (08/12) ras.py:RASTools.test8_nvram: PASS (2.25 s)
 (09/12) ras.py:RASTools.test9_ofpathname: FAIL: RAS: Failed commands are: ['ofpathname -l /vdevice/v-scsi@30000066/disk@8100000000000000:2'] (2.56 s)
 (10/12) ras.py:RASTools.test11_rtas_ibm_get_vpd: PASS (2.75 s)
 (11/12) ras.py:RASTools.test12_rtas_errd_and_rtas_dump: PASS (6.13 s)
 (12/12) ras.py:RASTools.test13_rtas_event_decode: PASS (3.94 s)
RESULTS    : PASS 11 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2020-12-31T14.12-2453f79/results.html
after patch
[root@ltcden13-lp2 ras]# avocado run ras.py
JOB ID     : 1e7d8c20e839384fe1716e5d5c24d75724cefacf
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2020-12-31T13.18-1e7d8c2/job.log
 (01/12) ras.py:RASTools.test1_set_poweron_time: PASS (1.35 s)
 (02/12) ras.py:RASTools.test2_sys_ident_tool: PASS (1.34 s)
 (03/12) ras.py:RASTools.test3_lsmcode: PASS (1.64 s)
 (04/12) ras.py:RASTools.test4_drmgr: PASS (4.61 s)
 (05/12) ras.py:RASTools.test5_lsprop: PASS (1.41 s)
 (06/12) ras.py:RASTools.test6_lsslot: PASS (1.56 s)
 (07/12) ras.py:RASTools.test7_lsvio: PASS (1.37 s)
 (08/12) ras.py:RASTools.test8_nvram: PASS (1.41 s)
 (09/12) ras.py:RASTools.test9_ofpathname: PASS (1.67 s)
 (10/12) ras.py:RASTools.test11_rtas_ibm_get_vpd: PASS (1.34 s)
 (11/12) ras.py:RASTools.test12_rtas_errd_and_rtas_dump: PASS (2.16 s)
 (12/12) ras.py:RASTools.test13_rtas_event_decode: PASS (1.38 s)
RESULTS    : PASS 12 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2020-12-31T13.18-1e7d8c2/results.html
JOB TIME   : 22.42 s

File: ras/ras.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def test9_ofpathname(self):
             self.run_cmd("ofpathname %s" % disk_name)
             of_name = self.run_cmd_out("ofpathname %s"
                                        % disk_name)
-            self.run_cmd("ofpathname -l %s" % of_name)
+            self.run_cmd("ofpathname -l %s" % of_name.split(':')[0])
```
Comment (harish-24) [inline ras/ras.py:205]: can we move this change to the previous line so that we can use it here?

**Comment by harish-24** [comment]
@debade33 Once job.log/summary is posted with the new change we are good.

**Comment by debade33** [comment]
After Patch
localhost:/home/OpTest/avocado-fvt-wrapper/tests/avocado-misc-tests/ras # avocado run ras.py
JOB ID     : 1eb22970fcb7a1eac5289e1f71f13c80bef40904
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2021-01-08T21.34-1eb2297/job.log
 (01/12) ras.py:RASTools.test1_set_poweron_time: PASS (1.79 s)
 (02/12) ras.py:RASTools.test2_sys_ident_tool: PASS (1.68 s)
 (03/12) ras.py:RASTools.test3_lsmcode: PASS (2.21 s)
 (04/12) ras.py:RASTools.test4_drmgr: PASS (5.11 s)
 (05/12) ras.py:RASTools.test5_lsprop: PASS (2.80 s)
 (06/12) ras.py:RASTools.test6_lsslot: PASS (1.94 s)
 (07/12) ras.py:RASTools.test7_lsvio: PASS (1.68 s)
 (08/12) ras.py:RASTools.test8_nvram: PASS (1.76 s)
 (09/12) ras.py:RASTools.test9_ofpathname: PASS (1.96 s)
 (10/12) ras.py:RASTools.test11_rtas_ibm_get_vpd: PASS (1.73 s)
 (11/12) ras.py:RASTools.test12_rtas_errd_and_rtas_dump: PASS (2.31 s)
 (12/12) ras.py:RASTools.test13_rtas_event_decode: PASS (1.71 s)
RESULTS    : PASS 12 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2021-01-08T21.34-1eb2297/results.html
JOB TIME   : 28.82 s

**Review by harish-24 — APPROVED** [review]
LGTM

**Pull request merged**